packages/mirta-testing/tests/integration/simulator.test.ts

```
@@ -0,0 +1,464 @@
+
+      expect(callback).toHaveBeenCalledWith({
+        topic: '/devices/myDevice/controls/myControl',
+        value: 42,
+      });
+
+    });
+
+    it('should trigger defineRule whenChanged when the value changes', () => {
+
+      const ruleFn = vi.fn();
+      defineRule({ whenChanged: 'myDevice/myControl', then: ruleFn });
```

Biome (2.4.4)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)